docs_src/microservices/security/edgex-secrets-config-proxy.md

```diff
@@ -0,0 +1,142 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+**edgex-secrets-config-proxy** – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-config** proxy [OPTIONS] SUBCOMMAND [ARG...]
+
+# DESCRIPTION
+
+**edgex-secrets-config-proxy** configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and relevant utility functions.
+
+These commands require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **-m** _/path/to/resp-init.json_ (optional)
+
+    Points to location of secret store master key. Defaults to /vault/config/assets/resp-init.json.
+
+# COMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will configure the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * --certificate _/path/to/certchain.pem_ (required)
+
+      Path to TLS leaf certificate (with optional appended certificate chain, not including a root certificate).
+
+    * --key _/path/to/private.pem_ (required)
+
+      Path to TLS private key.
+
+  * **adduser**
+
+    Create a proxy user using JWT public/private key authentication. Requires additional arguments:
+
+    * --oauth (unimplemented; reserved for future use)
```

Comment:
Is it not possible to create oauth tokens today? If so, why is this being left out?

Reply:
1. A JWT token can do everything that an OAuth token can do
2. The default EdgeX implementation is to use JWT.  (And in fact, if we changed it to OAuth, it would break the blackbox tests.)
3. If we decide to make Kong DB-less, OAuth tokens wouldn't be supported (OAuth requires a database).

Thus, I don't see a real need to continue to support OAuth going forward..

Reply:
Ok.  Since I don't want to commit to deprecating OAuth at this time, I will add the OAuth support in.